simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/mock/OcppMockServer.java

```diff
@@ -0,0 +1,151 @@
+package com.evbox.everon.ocpp.simulator.mock;
+
+import com.evbox.everon.ocpp.simulator.message.Call;
+import io.undertow.Undertow;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Objects;
+import java.util.function.Predicate;
+
+import static com.evbox.everon.ocpp.simulator.mock.ExpectedCount.once;
+import static io.undertow.Handlers.path;
+import static io.undertow.Handlers.websocket;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+/**
+ * WebSocket Ocpp mock server.
+ */
+@Slf4j
+public class OcppMockServer {
```

Comment:
I'd expect that you refactor/remove WebSocketServerMock, which was essentially the same thing, just with different DSL

Reply:
Afterwards. Otherwise MR gets big.

It has big differences. You have an API and verification which means you dont have to create private methods in the Tests. There are dedicated places for that.